util/reggen/bus_interfaces.py

```diff
@@ -96,11 +106,17 @@ def from_raw(raw: object, where: str) -> 'BusInterfaces':
                 if async_clk is not None:
                     device_async[name] = async_clk
 
+                if hier_path is not None:
+                    device_hier_paths[name] = hier_path
+                else:
+                    device_hier_paths[name] = 'u_reg'
```

Comment:
For next time, this sort of thing can be done a bit more cleanly with Python's equivalent of a ternary: `device_hier_paths_[name] = hier_path if hier_path is not None else 'u_reg'`. (Or, if you're being lazy, `device_hier_paths_[name] = hier_path or 'u_reg'`, but that handles empty strings differently so it's probably not quite right)

Reply:
Yeah I believe I did not use the ternary version here due to the line getting too long...